back/engines/commercial/flag_inappropriate_content/app/serializers/flag_inappropriate_content/web_api/v1/inappropriate_content_flag_serializer.rb

```diff
@@ -11,6 +11,19 @@ class WebApi::V1::InappropriateContentFlagSerializer < ::WebApi::V1::BaseSeriali
       end
     end
 
-    belongs_to :flaggable, polymorphic: true
+    # We specify the serializer(s) for the object(s), because if we just use polymorphic: true
+    # jsonapi-serializer will look for the serializer(s) in the same namespace as this serializer, which will fail.
+    belongs_to :flaggable, serializer: proc { |object|
+      case object
+      when Comment
+        ::WebApi::V1::CommentSerializer
+      when Idea
+        ::WebApi::V1::IdeaSerializer
+      when Initiative
+        ::WebApi::V1::InitiativeSerializer
+      else
+        raise ArgumentError, "Unknown flaggable type: #{object.class.name}"
+      end
+    }
```

Comment:
One of three cases where we need to give jsonapi-serializer a little help coping with namespaces for polymorphic record serializers.